### ISSUE TITLE
compile error

### ISSUE THREAD

**Issue by worker2345234** [zigtools/zls#119]
Hi,

I've got a compile error:
```
C:\temp\zls>zig build
.\src\main.zig:63:39: error: TODO: type coercion of anon struct literal to struct
    try std.json.stringify(reqOrRes, .{}, arr.outStream());
                                      ^
.\src\main.zig:194:13: note: called from here
    try send(types.Notification{
            ^
.\src\main.zig:127:75: note: called from here
fn publishDiagnostics(handle: DocumentStore.Handle, config: Config) !void {
                                                                          ^
.\src\main.zig:63:27: note: referenced here
    try std.json.stringify(reqOrRes, .{}, arr.outStream());
                          ^
.\src\main.zig:194:5: note: referenced here
    try send(types.Notification{
    ^
.\src\main.zig:971:9: note: referenced here
        try publishDiagnostics(handle.*, configFromUriOr(uri, config));
        ^
.\src\main.zig:1366:9: note: referenced here
        try processJsonRpc(&json_parser, buf, config, &keep_running);
        ^
zls...The following command exited with error code 1:
C:\Program Files\zig\zig.exe build-exe C:\temp\zls\src\main.zig --pkg-begin build_options C:\temp\zls\zig-cache\zls_build_options.zig --pkg-end --cache-dir C:\temp\zls\zig-cache --name zls --pkg-begin known-folders C:\temp\zls\src\known-folders\known-folders.zig --pkg-end --cache on

Build failed. The following command failed:
C:\temp\zls\zig-cache\o\2aCxu3sZT8J2_mEoCB25STpC_1PPEEAKtJ7REcBd8kGgMejnbfsoa52RIFD8FIqc\build.exe C:\Program Files\zig\zig.exe C:\temp\zls C:\temp\zls\zig-cache

C:\temp\zls>zig version
0.6.0+4302f276e
```

 I'm not sure if I need a newer or older compiler version.

**Comment by alexnask** [comment]
Hi, this was fixed in ziglang/zig@2bb3e1aff4976b2d04fb08a46d9221c77da0b204, you should be able to compile once you update your compiler.  
We only support master zig for compilation of zls at the moment, as is pointer out in the readme, you can download a master release [on the zig website](https://ziglang.org/download/).

**Comment by worker2345234** [comment]
Sorry and thx. I didn't expect my compiler to be to old because I downloaded the master approx. 1 week ago. I downloaded the master yesterday again, now everythink works.
